rocAL/rocAL/include/api/rocal_api_data_loaders.h

```diff
@@ -196,11 +196,13 @@ extern "C"  RocalImage  ROCAL_API_CALL rocalJpegCOCOFileSourcePartialSingleShard
                                                             unsigned shard_id,
                                                             unsigned shard_count,
                                                             bool is_output,
+                                                            std::vector<double>& area_factor, 
+                                                            std::vector<double>& aspect_ratio, 
+                                                            unsigned num_attempts,
                                                             bool shuffle = false,
                                                             bool loop = false,
-                                                            RocalImageSizeEvaluationPolicy decode_size_policy = ROCAL_USE_MAX_SIZE,
+                                                            RocalImageSizeEvaluationPolicy decode_size_policy = ROCAL_USE_USER_GIVEN_SIZE_RESTRICTED,
                                                             unsigned max_width = 0, unsigned max_height = 0,
-                                                            RocalFloatParam area_factor = NULL, RocalFloatParam aspect_ratio = NULL,
                                                             RocalFloatParam y_drift_factor = NULL, RocalFloatParam x_drift_factor = NULL );
```

Comment:
Is x_drift_factor and y_drift_factor needed as well? If not please get rid of those